### DISCUSSION TITLE
Vulkan compute without a renderer

### DISCUSSION THREAD

**Discussion by eliyam32** [KhronosGroup/Vulkan-Samples#1088] [General]
Hello, 
Im intrested to add Vulkan support to an embedded device.
It seems that Vulkan compute shaders are optionally an alternative to openCL. 

Vulkan has 2 parts : compute , rendering. 
is it possible to support Vulkan compute without  the rendering part ? 
are those parts easily separable ? 

Thanks,
Eliya

**Comment by SaschaWillems** [comment]
Yes, that's perfectly possible and usually called "headless". Rasterization, presentation and compute are separated in such a way that you can use them without any of the other. Our samples don't support this right now, but you can take a look at my [headless compute example](https://github.com/SaschaWillems/Vulkan/blob/master/examples/computeheadless/computeheadless.cpp) that uses only Vulkan's compute part.
